package.json

```diff
@@ -1,16 +1,69 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "3.2.0",
-	"publishConfig": {
-		"access": "public"
+	"version": "3.1.0",
```

Comment:
We will have to update the version  since we published 3.2.0 recently and I believe this PR was worked on before that

Reply:
Yup yup, Ill update that to what we have in `develop`. We can update the version before merging to `main`.